math/quadratic_equations_complex_numbers.cpp

```diff
@@ -0,0 +1,162 @@
+/**
+ * @file
+ * @brief Calculate quadratic equation with complex roots, i.e. b^2 - 4ac < 0.
+ * @see https://en.wikipedia.org/wiki/Quadratic_equation,
+ * https://en.wikipedia.org/wiki/Quadratic_equation#discriminant
```

Comment:
`@see` is used for files related to the current algorithm, not for Markdown links.
```suggestion
 * @brief Calculate [Quadratic Qquation](https://en.wikipedia.org/wiki/Quadratic_equation) with complex roots, i.e. b^2 - 4ac < 0.
```